{"name": "test_courier_login_invalid_credentials", "status": "passed", "description": "Ошибка при неверных логине и пароле", "start": 1729974275538, "stop": 1729974275588, "uuid": "dcf6cdae-43e0-43b9-8977-0c0d9b0406da", "historyId": "a813fb29600fed66279fd480400fc548", "testCaseId": "a813fb29600fed66279fd480400fc548", "fullName": "tests.orders.test_login_courier.TestLoginCourier#test_courier_login_invalid_credentials", "labels": [{"name": "parentSuite", "value": "tests.orders"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "WIN-975IU3SH152"}, {"name": "thread", "value": "55724-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.orders.test_login_courier"}]}